mailpoet/lib/EmailEditor/Engine/Renderer/Postprocessors/HighlightingPostprocessor.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\EmailEditor\Engine\Renderer\Postprocessors;
+
+/**
+ * This postprocessor replaces <mark> tags with <span> tags because mark tags are not supported across all email clients
+ */
+class HighlightingPostprocessor implements Postprocessor {
```

Comment:
Just an idea. Maybe we could do this also in a preprocessor by changing the innerHtml and saved content of the individual blocks. With this approach we wouldn't need to introduce postprocessors. On the other hand I don't see any other significant advantages. Maybe we could add some specific conditions per block if needed? What do you tnink?

Reply:
This is a good idea, but I'm afraid of different rendering implementations of each block. In my opinion, this approach is safer for us.